release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -863,7 +863,14 @@ Support for the Broadcom BCM57504 network interface controller is now available
 [id="ocp-4-14-networking-ovn-kubernetes-secondary-network"]
 ==== OVN-Kubernetes is available as a secondary network
 
-With this release, the {openshift-networking} OVN-Kubernetes network plugin allows the configuration of secondary network interfaces for pods. As a secondary network, OVN-Kubernetes supports both layer 2 switched and localnet switched topology networks.For more information about OVN-Kubernetes as a secondary network, see xref:../networking/multiple_networks/configuring-additional-network.adoc#configuration-ovnk-additional-networks_configuring-additional-network[Configuration for an OVN-Kubernetes additional network].
+With this release, the {openshift-networking} OVN-Kubernetes network plugin allows the configuration of secondary network interfaces for pods. As a secondary network, OVN-Kubernetes supports both layer 2 switched and localnet switched topology networks. For more information about OVN-Kubernetes as a secondary network, see xref:../networking/multiple_networks/configuring-additional-network.adoc#configuration-ovnk-additional-networks_configuring-additional-network[Configuration for an OVN-Kubernetes additional network].
+
+[id="ocp-4-14-global-ip-forwarding-network-disabled"]
+==== Global IP forwarding is disabled on OVN-Kubernetes based cluster deployments
+
+From this release, global IP address forwarding is disabled on OVN-Kubernetes based cluster deployments to prevent undesirable effects for cluster administrators with nodes acting as routers. OVN-Kubernetes now enables and restricts forwarding on a per-managed interface basis.
+
+You can control IP forwarding for all traffic on OVN-Kubernetes managed interfaces by using the `gatewayConfig.ipForwarding` specification in the `Network` resource. Specify `Restricted` to forward all traffic related to OVN-Kubernetes only. Specify `Global` to allow forwarding of all IP traffic. For new installations, the default is `Restricted`. Clusters upgraded to 4.14 are not affected by this change; the IP forwarding behavior remains unchanged and continues to be globally enabled. For more information about enabling IP forwarding globally, see xref:../networking/cluster-network-operator.adoc#nw-cno-enable-ip-forwarding_cluster-network-operator[Enabling IP forwarding globally].
```

Comment:
the link is "Enabling IP forwarding globally" but I don't see that actual title on the page - the page title is "Cluster Network Operator in OpenShift Container Platform"? It looks like it is actually configured in the "Cluster Network Operator configuration object" section: https://84003--ocpdocs-pr.netlify.app/openshift-enterprise/latest/networking/cluster-network-operator#nw-operator-cr-cno-object_cluster-network-operator so maybe link there?

Reply:
@kcarmichael08 I am going to get this merged first https://github.com/openshift/openshift-docs/pull/84041 I think that should solve the link issue